content/md/first-steps/manage-your-attributes.md

```diff
@@ -92,15 +92,16 @@ The attribute is now created, you **can add a translation for each enabled local
 
 ::: warning
 Please note that the **following attribute codes cannot be used** in Akeneo PIM:    
-`id`, `associationTypes`, `category`, `categories`, `categoryId`, `completeness`, `enabled`, `family`, `Family`, `groups`, `associations`, `products`, `scope`, `treeId`, `values`, `label`, `parent`.    
-We strongly recommend you to do not use any of these codes even if the letter case is different, for instance `Id`, `Groups`.
+`id`, `associationTypes`, `category`, `categories`, `categoryId`, `completeness`, `enabled`, `family`, `groups`, `associations`, `products`, `scope`, `treeId`, `values`, `label`, `parent`
+We strongly recommend you not to use any of these codes even if the letter case is different, for instance `Id`, `Groups`.
 :::
 
 ::: warning
 If you cannot access the attribute page or if you cannot add a new attribute, it is probably because you do not have the role permission to. Please refer to [Manage your user roles](/articles/build-your-user-roles.html) to learn more about roles.
 :::
 
-## Validation for the different attribute types
+## Validation & type specific parameters for the different attribute types
```

Comment:
not sure about the new title here... this part relates to specific attribute validation settings, maybe "## Validation settings for the different attribute types" ?

Reply:
As there is no "validation parameter" for reference entities but only a "type specific parameters", I wanted to specify it in the title for the user not to be lost. But maybe you're right and this title is not clear enough...

Reply:
Following our Slack discussion, I'd rather keep it as it is now and maybe add something somewhere else. Because we'll have to add all the "specific parameters" for all attributes, and not only entities. 
Maybe we'll have a dedicated page to explain how to use the ref entities.. in this case, we can put this section there. What do you think?

Reply:
Hello! We just took a look at the current help center and we've seen that for metrics for example, validation parameters & specific parameters are gathered in this part of the article. So I will put those related to entities also, but I can change the title by: "Parameters according to specific attributes." What do you think?